src/Pos/Block/Logic/Creation.hs

```diff
@@ -46,7 +46,8 @@ import           Pos.Core                   (Blockchain (..), EpochIndex,
                                              SlotId (..), SlotLeaders, crucialSlot,
                                              epochOrSlot, flattenSlotId, getEpochOrSlot,
                                              getSlotIndex, headerHash, mkLocalSlotIndex)
-import           Pos.Crypto                 (SecretKey, WithHash (WithHash))
+import           Pos.Crypto                 (SecretKey, WithHash (WithHash), Hash)
+import           Pos.Crypto.Hashing         (hash)
```

Comment:
I think `hash` can also be imported from `Pos.Crypto` (without `Hashing`), but it's not really important.